docs/development.rst

```diff
@@ -64,6 +64,15 @@ will override the defaults.
 Server Development
 ------------------
 
+All commits to the core python code must work in all versions of python 2.7 and above.
+Python code in plugins should also work in 2.7 and 3.x, but some plugins may depend
```

Comment:
You can probably safely assume it is compatible with 3.3, but 3.2 and below are not commonly supported among dependencies.